android/brave_java_resources.gni

```diff
@@ -10,6 +10,7 @@ brave_java_resources = [
   "java/res/anim/shake.xml",
   "java/res/color/crypto_wallet_radiobutton_color_selector.xml",
   "java/res/drawable-hdpi/add_funds.png",
+  "java/res/drawable-hdpi/background_bat.png",
```

Comment:
Would it be possible to use vector drawables for new icons ? instead of including images for different dpis, we can just include one image.

Reply:
Okay, I will do

Reply:
Fixed